packages/terra-demographics-banner/src/selectable-demographics-banner/SelectableDemographicsBanner.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import { KEY_SPACE, KEY_RETURN } from 'keycode-js';
+import Button from 'terra-button';
+import IconInfo from 'terra-icon/lib/icon/IconInformation';
+import DemographicsBannerDisplay from '../DemographicsBannerDisplay';
+import BackgroundTile from './_BackgroundTile';
+import SelectableTile from './_SelectableTile';
+import styles from './SelectableDemographicsBanner.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Application content to display in the banner.
+   */
+  applicationContent: PropTypes.node,
+  /**
+   * Age of the person.
+   */
+  age: PropTypes.string,
+  /**
+   * The persons date of birth
+   */
+  dateOfBirth: PropTypes.string,
+  /**
+   * The person's deceased date. Will display the banner as deceased if this value is provided
+   */
+  deceasedDate: PropTypes.string,
+  /**
+   * Gender of the Person
+   */
+  gender: PropTypes.string,
+  /**
+   * The persons gestational age.
+   */
+  gestationalAge: PropTypes.string,
+  /**
+   * Additional key value identifiers of a person's demographic information
+   */
+  // eslint-disable-next-line react/forbid-prop-types
+  identifiers: PropTypes.object,
+  /**
+   * @private
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
+   */
+  intl: intlShape.isRequired,
+  /**
+   * Full Name of the person
+   */
+  personName: PropTypes.string,
+  /**
+   * URL of Photo to display in the banner
+   */
+  photo: PropTypes.string,
+  /**
+   * The column layout in which to present the displays.
+   */
+  postMenstrualAge: PropTypes.string,
+  /**
+   * The persons preferred first name if they have one.
+   */
+  preferredFirstName: PropTypes.string,
+  /**
+   * Callback function triggered when info button is clicked.
+   */
+  infoButtonOnClick: PropTypes.func.isRequired,
+  /**
+   * Callback function triggered when selectable-tile is clicked.
+   */
+  selectableTileOnClick: PropTypes.func.isRequired,
+  /**
+   * Id for Info Button.
+   */
+  infoButtonId: PropTypes.string,
+  /**
+  * Id for Selectable Tile.
+  */
+  selectableTileId: PropTypes.string,
+};
+
+const defaultProps = {
+  applicationContent: null,
+  age: undefined,
+  dateOfBirth: undefined,
+  deceasedDate: null,
+  gender: undefined,
+  gestationalAge: null,
+  identifiers: {},
+  personName: undefined,
+  photo: undefined,
+  postMenstrualAge: null,
+  preferredFirstName: null,
+  infoButtonId: undefined,
+  selectableTileId: undefined,
+};
+
+const SelectableDemographicsBanner = ({
+  age,
+  dateOfBirth,
+  gender,
+  intl,
+  personName,
+  infoButtonOnClick,
+  selectableTileOnClick,
+  infoButtonId,
+  selectableTileId,
```

Comment:
All of the first class props should be spread here to differentiate from the custom props.

Reply:
[updated](https://github.com/cerner/terra-core/blob/b7897458163ba5b8e2bd88cfc5faa93b14d56aff/packages/terra-demographics-banner/src/SelectableDemographicsBanner.jsx#L100-L115)